composer.lock

```diff
@@ -150,7 +150,7 @@
             "version": "v3.4.1",
             "source": {
                 "type": "git",
-                "url": "https://github.com/twbs/bootstrap.git",
+                "url": "git@github.com:twbs/bootstrap.git",
```

Comment:
This change is not good, we need to keep til url as https://github.com/twbs/bootstrap.git - not git@github.com....